promql/quantile.go

```diff
@@ -505,14 +505,14 @@ func ensureMonotonicAndIgnoreSmallDeltas(buckets buckets, tolerance float64) (bo
 	return forcedMonotonic, fixedPrecision
 }
 
-// quantile calculates the given quantile of a vector of samples.
+// Quantile calculates the given Quantile of a vector of samples.
 //
 // The Vector will be sorted.
 // If 'values' has zero elements, NaN is returned.
 // If q==NaN, NaN is returned.
 // If q<0, -Inf is returned.
 // If q>1, +Inf is returned.
-func quantile(q float64, values vectorByValueHeap) float64 {
+func Quantile(q float64, values vectorByValueHeap) float64 {
```

Comment:
Doesn't this cause trouble because `vectorByValueHeap` is not exported?

Reply:
Looks like we don't need this one. Just HistogramQuantile and BucketQuantile.